pkg/app/mssql.go

```diff
@@ -0,0 +1,251 @@
+package app
+
+import (
+	"context"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/pkg/errors"
+	appsv1 "k8s.io/api/apps/v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+
+	crv1alpha1 "github.com/kanisterio/kanister/pkg/apis/cr/v1alpha1"
+	"github.com/kanisterio/kanister/pkg/field"
+	"github.com/kanisterio/kanister/pkg/kube"
+	"github.com/kanisterio/kanister/pkg/log"
+)
+
+const (
+	mssqlWaitTimeout = 2 * time.Minute
+)
+
+type MssqlDB struct {
+	cli        kubernetes.Interface
+	namespace  string
+	name       string
+	deployment *appsv1.Deployment
+	service    *v1.Service
+	pvc        *v1.PersistentVolumeClaim
+	secret     *v1.Secret
+	dbUserName string
+	dbPass     string
+}
+
+func NewMssqlDB(name string) App {
+	return &MssqlDB{
+		name: name,
+		// These values are hard coded while creating secret
+		dbUserName: "sa",
+		dbPass:     "MyC0m9l&xP@ssw0rd",
+	}
+}
+
+func (m *MssqlDB) ConfigMaps() map[string]crv1alpha1.ObjectReference {
+	return nil
+}
+
+func (m *MssqlDB) Secrets() map[string]crv1alpha1.ObjectReference {
+	return map[string]crv1alpha1.ObjectReference{
+		"mssql": {
+			Kind:      "Secret",
+			Name:      m.name,
+			Namespace: m.namespace,
+		},
+	}
+}
+
+func (m *MssqlDB) Init(ctx context.Context) error {
+	cfg, err := kube.LoadConfig()
+	if err != nil {
+		return err
+	}
+
+	m.cli, err = kubernetes.NewForConfig(cfg)
+	return err
+}
+
+func (m *MssqlDB) Install(ctx context.Context, namespace string) error {
+	m.namespace = namespace
+	secret, err := m.cli.CoreV1().Secrets(namespace).Create(ctx, m.getSecretObj(), metav1.CreateOptions{})
+	if err != nil {
+		return err
+	}
+	log.Print("Secret with name "+secret.Name+" created successfully", field.M{"app": m.name})
```

Comment:
Lets use `log.Info().Print()` as we are doing in other apps.
Lets make `secret.Name` as a field element.
Ref: https://github.com/kanisterio/kanister/blob/master/pkg/app/postgresql.go#L151
```suggestion
	log.Info().Print("Secret created successfully", field.M{"app": m.name, "secret":secret.Name})
```
Please follow the same format for other logs. The variables can be moved to field elements.

Reply:
In one of previous comment you suggested to move logs to debug. That's why I shifted to Debug